Remotion/ObjectBinding/Core/BusinessObject.cs

```diff
@@ -29,12 +29,12 @@ public abstract class BusinessObject : IBusinessObject
 
```

Comment:
GetProperty returns nullable. Please also check hierarchy for consisntent nullability